cmd/mountlib/mount.go

```diff
@@ -22,6 +25,30 @@ var (
 	ExtraFlags         *[]string
 )
 
+// Check is folder is empty
+func checkMountEmpty(mountpoint string) error {
+	fp, fpErr := os.Open(mountpoint)
+	defer fs.CheckClose(fp, &fpErr)
```

Comment:
That defer fs.CheckClose needs to go after the if fpErr != nil check I just noticed.  fp will be nil if fpErr != nil.